lukefi/metsi/forestry/cross_cutting/cross_cutting.py

```diff
@@ -11,9 +13,8 @@
     TreeSpecies.SILVER_BIRCH: "birch"
 }
 
-ZERO_DIAMETER_TREE_TIMBER_GRADE = 3 # = energy wood
-ZERO_DIAMETER_TREE_VOLUME = 0.000045 # m3
-ZERO_DIAMETER_TREE_VALUE = 20 #€/m3
+ZERO_DIAMETER_DEFAULTS = ([3], [0.000045], [20])
```

Comment:
It is somewhat difficult to say what these values stand for. By looking at the docstring of `cross_cut` it becomes clearer, but at least with the first element in the tuple it is difficult to say that it is a timber grade value representing energy wood.

Reply:
I'll return the commentary and add reference to the real source too